python/tk_multi_publish2/api/manager.py

```diff
@@ -276,14 +277,15 @@ def all_tasks_generator(publish_tree):
 
         :param task_generator: A generator of :class:`~PublishTask` instances.
 
-        :returns: A list of items with tasks that failed to validate.
+        :returns: A list of tuples of (:class:`~PublishTask`,
+            optional :class:`Exception`) that failed to validate.
```

Comment:
What does optional mean in this case? If no exception was raised, is `(task, None)` returned? Worth clarifying!

Reply:
Will clarify!